R/methods_tidymodels.R

```diff
@@ -8,10 +8,21 @@ get_predict.model_fit <- function(model, newdata, type = NULL, ...) {
     return(out)
 }
 
+#' @include get_predict.R
+#' @rdname get_predict
+#' @keywords internal
+#' @export
+get_predict.workflow <- get_predict.model_fit
```

Comment:
`workflow`s and `model_fit`s have type-stable prediction! https://tidymodels.github.io/model-implementation-principles/model-predictions.html#return-values

Seems like this is a TODO already, but predictions will have column names different (but deterministically) than `".pred"` in the classification case. If you just need hard class predictions, that column will be `".pred_class"`. Class probability columns are named w.r.t. the levels of the outcome.